vms/registration/templates/registration/signup_volunteer.html

```diff
@@ -42,7 +42,7 @@ <h2>{% trans "Sign Up" %}</h2>
                     {% trans "Password" %}<span class="asteriskField">*</span>
                 </label>
                 <div class="controls">
-                    <input id="id_password" class="textinput textInput form-control" type="password" placeholder="{% blocktrans %}Password{% endblocktrans %}" name="usr-password">
+                    <input id="id_password" class="textinput textInput form-control" type="password" placeholder="{% blocktrans %}Password ( password length should be at least 8 characters long){% endblocktrans %}" name="usr-password">
```

Comment:
Add maximum length error message too for passwords and as @tapasweni-pathak said an error message accordingly and not placeholder text.